pkgs/development/libraries/rutabaga_gfx/default.nix

```diff
@@ -36,6 +36,6 @@ stdenv.mkDerivation (finalAttrs: {
     description = "cross-platform abstraction for GPU and display virtualization";
     license = licenses.bsd3;
     maintainers = with maintainers; [ qyliss ];
-    platforms = platforms.darwin ++ platforms.linux;
+    platforms = gfxstream.meta.platforms;
```

Comment:
Technically rutabaga doesn't require gfxstream (and therefore aemu), so I wonder whether we could omit that support on those platforms.  (Wayland forwarding should still work.)